src/Keboola/DbExtractor/Application.php

```diff
@@ -35,7 +35,8 @@ public function __construct(array $config, Logger $logger, array $state = [])
         $this['logger'] = $logger;
 
         $this['extractor_factory'] = function () use ($app) {
-            return new ExtractorFactory($app['parameters'], $app['state']);
+            $configData = $app->config->getData();
+            return new ExtractorFactory($configData['parameters'], $app['state']);
```

Comment:
myslim se ze tady jeste neni definovane `app->config` ne?  It doesn't get added until the build config call?

Reply:
je definovaný - funkce `$this['extractor']` se volá až po spuštění `run()` ještě bych mohl pro přehlednost přesunout `$this->buildConfig($config);` nad definici těchto funkcí?

Reply:
uz mame `$this->config` definovany tak muzem to pouzivat tady jako `$configData = $this->config->getData()`?  Ale asi je to jedno.